todolist.js

```diff
@@ -15,6 +15,14 @@
       paragraph.innerText = domElems.inputField.value;
       taskName = domElems.inputField.value;
 
+      const delTaskBtn = document.createElement('button');
+      delTaskBtn.innerHTML = 'Delete';
+      delTaskBtn.addEventListener('click', function () {
```

Comment:
keep it function outside from the addEventListener, so it is better to resolve the bug

Reply:
Okay, it has been resolved.